plugins/import/etherscan.ts

```diff
@@ -75,7 +78,26 @@ export function getEtherscanApiKey(network: string): string {
   return apiKey;
 }
 
-export async function get(url, data) {
-  const res = (await axios.get(url, { params: data }))['data'];
-  return res;
+export async function get(url, data, network = '') {
+  try {
+    let res;
+    // TODO remove this condition
+    if (network === 'linea-goerli') {
+      res = (
+        await axios.get(url, {
+          params: data,
+          auth: {
+            username: 'eslineastage',
+            password: 'QW$eKsy3J%~v'
+          }
+        })
+      )['data'];
```

Comment:
Note: Remember to remove.

Reply:
Good catch thank you, completely forgot about that TODO

Reply:
Following up on this again, did you want to remove this before merging?